### PR TITLE
feauture-9[create-a-hero-section]

### PR DESCRIPTION
**Pull request by Salamah-Jimoh**
## Description

 This PR adds a new component for displaying a hero for users 

 Closes#9

## Acceptance Criteria

 The section should be about 80% of the screen height
 The section should have a background color HEX #f5f5f5 the whole with
 The content should cover 80% of the width
 The text content contains a short header followed by an paragraph 3-4 lines long (lorem ipsum)
 Use semantic HTML
 Use default styling for the text, no CSS needed
 Use a random image/graphic

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|  ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

File: components/hero/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...6MCGVhcEHUWMZL6LJX+Jldq/iBhrFdz2h090ED1b9rzmW08WKtQuF3QeF7+ptnLXxZsH+HqkoCaTZqTc7t/5CenA8R5zOxZNvs5c2TI/jIKCCCAgLQSQgjCR5AYkhWPJHFEaRukcWoOok2mKOiRsRX4DzMTXr3yGnzgSnbM+Q+sMY1thEpT4n04x4LxhqVGZzP3xiHqXytlM7bM2IvFiJ8oodoRBynUKkMNQQR3E6BsvHh0BHETng7S32Fj9xgrflJ9LzLRXDLjKmb9MQRYjUZiXG+GUOuhF5mFqka58jzEtNk4rModDmvQ8R5x3G0ei1yiFtfZy16bI3dTxVuBnMsRRam7I4symx/YjpxnX2SYfx9g90pWA19xj/qX/d8IjWjkz47jy9ozNNxLXB0najVNIsrKAwKEq1xc5EZ8JSIQZf7F2h7JLW1zv8JFunZxxVMqsL4yxiZe1Ljk4DfHX4y1o/iFVtZ6SN1VmT53mY2vh92oSB7re8PPX4yvnXF2rR1RySXTNdtTxaKyMvsyLi2bgj5ZzJmJghbsM8kp1yfQII9SWCLZOxtDJSmRBHlNoJIWRIUxl3vC3ico9TS1+cToAES1idYtz/bz+zCYxA084O9gYFhssJYoCEUCxQGUSRDBgMKEUmRiVhrAA1GxtqggUqht+hz/AOJ+ss6jlT8iJihnHUxbqLBzbgDnaOpHVh8njqR1PZm0BUWxI3xr16yp8cIDhn6FSPJh9TMPQ2w6MGBzHKSds+JqmIT2bKqjK5BNzY3+kZtNFp5scouikorcgS7ROEqdnr73kZd07Tlyd0cJC2vhSae9+nPyORmbm/qqrYete2SMR6ZTAS+H8aHiCOU6d4gCTEUKLnWUboLY4ihRcwpEq1SxhROAKGwIsREUsdhY9Ta0fBkdTFI0SSFHjCAy84neikOsCAxLCKVoLwW5QihiGIkQwYDCul4pYkwxrAAeWEYLwATUYZcxDR1hGQhJsJgol4Aanyk01DzjVGnuqBHCJzydysYaxBLKRfUShIztNDuytxlMK29zzlsMq0GLGUUKLnWMVKhJuYHe8RLpe2MkCCCCEIIBBBMYdSKEEEViMcHCKp6wQRACn1MSIIIRQzCWCCAI5DOsEEAo40Cw4ITCKkewoyhwRZdDokLHBBBIGFCVO2vzL2PzgglMX5Bj2VkEEE6hwQQQTGP/2Q=="
```
Comment (YuriDevAT) [inline components/hero/Hero.jsx:12]: For image, please create a folder in the public folder called images and store it there. Import the image then to make the component more readable.

File: components/hero/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styles from './hero.module.css'
+const Hero = () => {
```
Comment (YuriDevAT) [inline components/hero/Hero.jsx:3]: For better readability and consistency keep a space between imports and the beginning of the arrow function.

File: components/hero/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <section className ={styles.hero}>
+        <div className={styles.heroContent}>
```
Comment (YuriDevAT) [inline components/hero/Hero.jsx:6]: For consistency use space 2 for insets throughout the whole component (and other files you may created).

File: components/hero/Hero.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            <div className={styles.textContent}>
+                <h2>Heading</h2>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Veniam dolorum perspiciatis repudiandae quidem ex! Fugiat ea cum nam eos consectetur quo delectus debitis, aspernatur consequuntur. </p>
+            </div>    {/*textContent div ends */}
```
Comment (YuriDevAT) [inline components/hero/Hero.jsx:10]: Remove unnecessary comments throughout your created files to ensure clean code.

File: components/hero/hero.module.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
+/** Hero styles **/
+.hero{
+    margin-top:2rem;
+    background-color: #f5f5f5;
+    height:80vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.heroContent{
```
Comment (YuriDevAT) [inline components/hero/hero.module.css:10]: For better readability add a empty line between each style.

File: components/hero/hero.module.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
+/** Hero styles **/
+.hero{
```
Comment (YuriDevAT) [inline components/hero/hero.module.css:2]: For better readability add a space between class and arrow brackets.

File: components/hero/hero.module.css [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    margin-top:2rem;
```
Comment (YuriDevAT) [inline components/hero/hero.module.css:3]: As mentioned in the Hero.jsx file keep a space inset from two throughout the files.

File: package-lock.json
Comment (YuriDevAT) [inline package-lock.json:1]: Since you did not install a new dependency do not add the package-lock.json file to the PR.

**Review by YuriDevAT — CHANGES_REQUESTED** [review]
You did a great job, @Salamah 👍🏽 I added some comments on how you can improve your code for a great DX (developer experience). If you have any questions let me know 😊

**Comment by Salamah-Jimoh** [comment]
Alright. I will make the required corrections. Thank you

**Pull request merged**